features/admin.connections.v1/components/edit/forms/general-details-form.tsx

```diff
@@ -285,6 +286,19 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                     onSubmit={ (values: GeneralDetailsFormValuesInterface): void => {
                         updateConfigurations(values);
                     } }
+                    validate={ (values: GeneralDetailsFormValuesInterface) => {
+                        const errors: Partial<Record<keyof GeneralDetailsFormValuesInterface, string>> = {
+                            image: undefined
+                        };
+
+                        if (values?.image && values.image !== " " && !ImageUtils.isValidImageExtension(values?.image)) {
+                            errors.image = t(
```

Comment:
```suggestion
                            errors?.image = t(
```

Reply:
We don't need optional chaining here as both `errors` object and `image` property are guaranteed to be available at this point.